addon/components/upf-stat.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  classNames: ['upf-stat'],
+
+  name: null,
+  data: null,
+
+  progressValue: null,
```

Comment:
This data should be calculated before. It's not better to have a maxValue and compute the progress in the component ?